http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterpSpec.scala

```diff
@@ -313,18 +305,14 @@ class AppServiceInterpTest extends AnyFlatSpec with AppServiceInterpSpec with Le
     val mockAuthProvider = mock[LeoAuthProvider[IO]]
     when(mockAuthProvider.hasPermission(any, any, any)(any, any)).thenReturn(IO.pure(true))
     when(mockAuthProvider.lookupOriginatingUserEmail(any)(any)).thenReturn(IO.pure(userInfo.userEmail))
-    when(mockAuthProvider.notifyResourceCreated(any[AppSamResourceId], any, any)(any, any, any)).thenReturn(IO.unit)
-    when(mockAuthProvider.notifyResourceDeleted(any[AppSamResourceId], any, any)(any, any)).thenReturn(IO.unit)
     val publisherQueue = QueueFactory.makePublisherQueue()
     val appService = makeInterp(publisherQueue, authProvider = mockAuthProvider)
     val res = appService
       .createApp(userInfo, cloudContextGcp, appName, appReq)
       .attempt
       .unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
     res.swap.toOption.get.getMessage shouldBe "Disk is not formatted yet. Only disks previously used by galaxy/cromwell/rstudio app can be re-used to create a new galaxy/cromwell/rstudio app"
-
-    verify(mockAuthProvider).notifyResourceCreated(any[AppSamResourceId], any, any)(any, any, any)
-    verify(mockAuthProvider).notifyResourceDeleted(any[AppSamResourceId], any, any)(any, any)
+    // TODO verify SamService
```

Comment:
What do you mean here?

Reply:
Ah oops, this test used to verify that the Sam resource was created and removed on errors. Looks like I removed those assertions and didn't replace them. Will fix.

Reply:
Fixed